### DISCUSSION TITLE
search in attributes

### DISCUSSION THREAD

**Discussion by plotek** [freeplane/freeplane#731] [Help] [answered]
Hello !

I'm using the latest Freeplane version 'Freeplane-Setup-with-Java-1.10.4' with Java 15.0.7.
I can't search for attributes in the filter bar!
How to proceed, please?
Thanks in advance to anyone willing to help me.

En français : j'utilise la dernière version de Freeplane "Freeplane-Setup-with-Java-1.10.4" avec Java 15.0.7.
Je ne parviens pas à lancer des recherches sur les attributs dans la barre de filtre !
Comment procéder, svp ?

Merci à ceux qui voudront bien m'aider.

**Comment by abc16361** [comment]
You can try the steps on the photo to see whether it works for you. There are more options on step 2. I don't use them much, but you can try.
![AttributeFilter](https://user-images.githubusercontent.com/101362122/194686906-d55b3a10-d7a6-47ec-b8c0-def789cc1406.png)

**Comment by plotek** [comment, marked as answer]
Thanks a lot. There's a scrollbar on the right in the 'Attribute name or value' box (notably) that I couldn't see.
Thanks again and have a great day.

Merci beaucoup. Il y a un ascenseur à droite dans la zone "Attribute name or value (notamment) que je ne voyais pas.
Merci à nouveau et très bonne journée.
